cmake/modules/RootNewMacros.cmake

```diff
@@ -1079,6 +1079,22 @@ function(ROOT_ADD_TEST test)
 endfunction()
 
 #----------------------------------------------------------------------------
+# function ROOT_ADD_GTEST(<testsuite> <name> LIBRARIES)
+#
+function(ROOT_ADD_GTEST test_suite test_name)
```

Comment:
I would request that we do this from day 0. Else it'll be a mess, with people adding their files by hand, until you find the time to do it.